ultimatejobweb/migrations/test_data_jobs1.py

```diff
@@ -0,0 +1,39 @@
+from django.db import migrations, transaction
+from ultimatejobweb.models import Job, Company
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('ultimatejobweb', '0001_initial'),
+    ]
+
+    def generate_data(apps, schema_editor):
+        company1 = Company(1, '"https://careers-redhat.icims.com/jobs/search?ss=1&in_iframe=1&searchKeyword=',
+                           'logo1', 'function1')
+        company2 = Company(2, 'https://www.amazon.jobs/en/', 'logo2', 'function2')
+
```

Comment:
Neither of these 2 companies are being saved - you didn't use .save()
You are also creating two companies in the for loop below.

Reply:
@omernu Please consider working with objects
.....
        companies = [
          (company1, 'https://careers-redhat.icims.com/jobs/search?ss=1&in_iframe=1&searchKeyword=',
           'logo1', 'function1'),
          (company2, 'https://www.amazon.jobs/en/', 'logo2', 'function2'),
        ]
......
            for company, job_title, description_url in jobs:
                company_n = Company.objects.get(company_name=company)
                Job(company=company_n, job_title=job_title, description_url=description_url).save()

Reply:
Lets get this resolved so we can merge the PR